device_doctor/lib/src/android_device.dart

```diff
@@ -164,6 +165,24 @@ class AndroidDeviceDiscovery implements DeviceDiscovery {
     }
     return healthCheckResult;
   }
+
+  @visibleForTesting
```

Comment:
Can we add some docs on what is this checking?

Reply:
Added.